rum-spa-test.html

```diff
@@ -52,59 +52,16 @@ <h2>das da stuff</h2>
       rg4js('enableCrashReporting', true);
       rg4js('enablePulse', true);
       rg4js('options', {
-        automaticPerformanceCustomTimings: true
+        automaticPerformanceCustomTimings: true,
+        trackCoreWebVitals: true
       });
 
-      rg4js('trackEvent', {
-        type: 'customTiming',
-        name: 'customTiming',
-        duration: 1200,
-      });
-
-      window.performance.mark('pageNavigation');
-
-      setTimeout(() => {
-        window.performance.measure('testPageNavigation', 'pageNavigation');
-
-
-        rg4js('trackEvent', {
-          type: 'pageView',
-          path: '/menu'
-        });
-
-        setTimeout(() => {
-          rg4js('trackEvent', {
-            type: 'pageView',
-            path: '/order'
-          });
-
-          setTimeout(() => {
-            rg4js('trackEvent', {
-              type: 'pageView',
-              path: '/login'
-            });
-
-            rg4js('setUser', {
-              identifier: emails[Math.floor(Math.random() * (emails.length - 1))]
-            });
-
-            setTimeout(() => {
-              rg4js('trackEvent', {
-                type: 'pageView',
-                path: '/order'
-              });
-            }, 50);
+      setTimeout(function () {
+        var button = document.getElementById("btn");
+        button.click();
+      }, 1000);
 
-            setTimeout(() => {
-              rg4js('trackEvent', {
-                type: 'pageView',
-                path: '/success'
-              });
-            }, 250);
-          }, 250);
-        }, 250);
-      }, 250);
```

Comment:
Do we not want these? 

I think it may also be good to have unit tests for the new CWV similar to what we've done for custom timings [here](https://github.com/MindscapeHQ/raygun4js/blob/master/tests/specs/v2/customTiming.js), where we are checking that the payloads are sent

Reply:
I will bring these back.
In terms of the unit tests they cannot be done in this way as cwv's are not tracked by the `web-vitals` library when chrome is running headless, I previously tried this with Ben and we came to this conclusion. Instead I have written unit tests for the interior functions instead of testing whether the library itself does it's job.

Reply:
Ah that's a shame, would've been good to ensure the payloads are sent to our API 😞